From 101c500f7b2431c14d1bf8f27c8fe61a784b391b Mon Sep 17 00:00:00 2001
From: Bogdan Hamciuc <bogdan.hamciuc@freescale.com>
Date: Tue, 6 Oct 2015 17:18:18 +0300
Subject: [PATCH 133/245] net/fsl: Enable XGMAC_MDIO on LS2085A SoC

XGMAC_MDIO is defined under NET_VENDOR_FREESCALE, which historically
was enabled on some Power platforms produced by Freescale. But some
newer, ARM-based, SoCs also have it, LS2085A being an example.

Signed-off-by: Bogdan Hamciuc <bogdan.hamciuc@freescale.com>
---
 drivers/net/ethernet/freescale/Kconfig | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/drivers/net/ethernet/freescale/Kconfig b/drivers/net/ethernet/freescale/Kconfig
index d0c0ad7..bd739db 100644
--- a/drivers/net/ethernet/freescale/Kconfig
+++ b/drivers/net/ethernet/freescale/Kconfig
@@ -7,7 +7,8 @@ config NET_VENDOR_FREESCALE
 	default y
 	depends on FSL_SOC || QUICC_ENGINE || CPM1 || CPM2 || PPC_MPC512x || \
 		   M523x || M527x || M5272 || M528x || M520x || M532x || \
-		   ARCH_MXC || ARCH_MXS || (PPC_MPC52xx && PPC_BESTCOMM)
+		   ARCH_MXC || ARCH_MXS || (PPC_MPC52xx && PPC_BESTCOMM) || \
+		   ARCH_FSL_LS2085A
 	---help---
 	  If you have a network (Ethernet) card belonging to this class, say Y
 	  and read the Ethernet-HOWTO, available from
@@ -67,8 +68,8 @@ config FSL_XGMAC_MDIO
 	select PHYLIB
 	select OF_MDIO
 	---help---
-	  This driver supports the MDIO bus on the Fman 10G Ethernet MACs, and
-	  on the FMan mEMAC (which supports both Clauses 22 and 45)
+	  This driver supports the MDIO bus on the Fman 10G Ethernet MACs and
+	  on mEMAC (which supports both Clauses 22 and 45)
 
 config UCC_GETH
 	tristate "Freescale QE Gigabit Ethernet"
-- 
2.1.0.27.g96db324

